formats/polus-feather-to-tabular-plugin/plugin.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Feather to Tabular",
+  "version": "0.1.0",
+  "title": "Feather to Tabular",
+  "description": "WIPP plugin to converts Feather file format to Tabular Data.",
+  "author": "Kelechi Nina Mezu (nina.mezu@nih.gov)",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/PolusAI/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "",
+  "containerId": "polusai/feather-to-tabular-plugin:0.1.0",
+  "inputs": [
+    {
+      "name": "filePattern",
+      "type": "string",
+      "description": "Filename pattern to convert feather file to",
+      "required": true
+    },
+    {
+      "name": "inpDir",
+      "type": "genericData",
+      "description": "Input image collection to be processed by this plugin",
+      "required": true
+    }
+  ],
```

Comment:
We should add another input called `format`. It should be an enum with the options `.pq` and `.csv`, then you can use that as the file extension in your code.